yt_dlp/extractor/nytimes.py

```diff
@@ -59,51 +48,26 @@ def get_file_size(file_size):
                 subtitles = self._merge_subtitles(subtitles, m3u8_subs)
             elif ext == 'mpd':
                 continue
```

Comment:
do you know why the old code was skipping dash manifest extraction? is it problematic in some way? should we do this?
```suggestion
                self.report_warning('Skipping DASH formats', video_id)
```

Reply:
I removed this check. 
None of the test I currently have in the extractor return dash as format option. 
Maybe there was a limitation at the time the original extractor was written.

Reply:
I think we still need to check, or else we could be appending an mpd URL as if it's a direct http format

Reply:
Fair enough. I reverted the change and added the ``report_warning`` as you suggested.

Reply:
Or we could add ``self._extract_mpd_formats(video_url, video_id,)``.